server/cli/launch.py

```diff
@@ -65,10 +64,12 @@ def launch(data, layout, diffexp, title, verbose, debug, obs_names, var_names,
         file_parts = splitext(basename(data))
         title = file_parts[0]
 
-    if listen_all:
-        host = "0.0.0.0"
+    if host:
+        host = host
+        flask_host = "0.0.0.0"
```

Comment:
if the user is expected to provide the local ip address, you might as well bind only to that ip address (caveat: NAT is the exception, but if the user is behind a NAT, they have a lot of other hoops to jump through).

Reply:
agree...